src/xdr/Stellar-ledger.x

```diff
@@ -111,6 +136,8 @@ case LEDGER_UPGRADE_MAX_TX_SET_SIZE:
     uint32 newMaxTxSetSize; // update maxTxSetSize
 case LEDGER_UPGRADE_BASE_RESERVE:
     uint32 newBaseReserve; // update baseReserve
+case LEDGER_UPGRADE_FLAGS:
+    uint32 newFlags; // update flags
```

Comment:
I talked to @MonsieurNicolas about this and we agreed that the behavior in this PR is safe and easy enough to use. I don't think we need to care about scenarios where validators vote for different flags. They are still voting on different states, so choosing a subset of the flags isn't safe.

In the second example where Node A votes for a different flag, this will not matter unless other nodes choose to vote along with it, so it doesn't sound like it would be an issue.